data_processing_pipeline/2_process/src/process_urban_areas.R

```diff
@@ -9,7 +9,7 @@ process_digitized_urban_extent <- function(target_name, digitized_shp_fn, proj_s
 process_modern_urban_extent <- function(target_name, urban_extent_shp_fn, state_nm, proj_str) {
   state_sf <- st_as_sf(maps::map("state", state_nm, plot = FALSE, fill = TRUE)) %>% 
     st_transform(crs = proj_str) %>%
-    lwgeom::st_make_valid() %>% 
+    st_make_valid() %>% 
```

Comment:
st_make_valid is an `sf` function

Reply:
is it? it only works for me from `lwgeom`